src/defines.asm

```diff
@@ -685,6 +685,7 @@ endif
 !ACTION_MANAGE_PRESETS      = #$0028
 
 !TOP_DISPLAY_VANILLA = #$0002
+!TOP_DISPLAY_VANILLA_8BIT = #$02
```

Comment:
These _8BIT defines aren't necessary. They can be shortened with .b on the opcode